README.md

```diff
@@ -12,21 +12,36 @@ This site is built on SvelteKit.
 
 ## Prerequisite
 
-Install Node - https://nodejs.org/en
-
-Install pnpm - https://pnpm.io/installation
-
-Install mysql
+* Install Node - https://nodejs.org/en
+* Install pnpm - https://pnpm.io/installation
+  * If you are on a Mac, there is an issue with the curl install.
+  * Preferably, use homebrew to install:
+    * `brew install pnpm`
+* Install mysql or use docker with the provided `docker-compose.yml` file.
 
 This site uses MySQL via Prisma, so you will need a valid MySQL connection string.
 
 ## Getting Started
 
 1. Read Prerequisites above ^^ before starting
-1. Copy `.env.example` to `.env` and specify env variables (needs at least `DATABASE_URL`)
-1. Run -> `pnpm preheat`
-1. Run -> `pnpm dev`
-1. Visit `http://localhost:5173`
+2. Copy `.env.example` to `.env` and specify env variables (needs at least `DATABASE_URL`)
+  * If using Docker, add the following at the top of your `.env` file to share variables with the `docker-compose.yml` (replace the existing DATABASE_URL with the one below)
+
+```sh
+DOCKER=true # this is required to make sure the seed commands are run within the container
+DATABASE_HOST=localhost
+DATABASE_PORT=3306
+DATABASE_USER=syntax
+DATABASE_PASSWORD=syntax
+DATABASE_NAME=syntax
+DATABASE_ROOT_PASSWORD=syntax
+DATABASE_URL=mysql://${DATABASE_USER}:${DATABASE_PASSWORD}@${DATABASE_HOST}:${DATABASE_PORT}/${DATABASE_NAME}
+```
+3. If using docker, run -> `docker compose up`
```

Comment:
I've never figured out how to get markdown numbers to continue from the previous numbers if there is a block in between them 🤔 

I just hard coded the incrementing numbers for now.

Reply:
Hey there, 
just some Syntax listener here.

I think this might be one way you could increment numbers in markdown with blocks between them:

```md

1. Read Prerequisites above ^^ before starting
1. Copy `.env.example` to `.env` and specify env variables (needs at least `DATABASE_URL`)
    * If using Docker, add the following at the top of your `.env` file to share variables with the `docker-compose.yml` (replace the existing DATABASE_URL with the one below)

      ```sh
      DOCKER=true # this is required to make sure the seed commands are run within the container, any value other than "true" is considered false
      DATABASE_HOST=localhost
      DATABASE_PORT=3306
      DATABASE_USER=syntax
      DATABASE_PASSWORD=syntax
      DATABASE_NAME=syntax
      DATABASE_ROOT_PASSWORD=syntax
      DATABASE_URL=mysql://${DATABASE_USER}:${DATABASE_PASSWORD}@${DATABASE_HOST}:${DATABASE_PORT}/${DATABASE_NAME}
      ```
1. If using docker, in a separate tab run -> `docker compose up`
1. Run -> `pnpm preheat`
1. Run -> `pnpm db:push`
1. Run -> `pnpm dev`
1. Visit `http://localhost:5173`
```
(adding a little indentation for the bullet point and code block I think did it!)

Reply:
Ehhh good call. That seems to work.